bin/fastatic

```diff
@@ -19,8 +20,12 @@ fastatic({
 		dest: program.dest,
 		logLevel: program.logLevel
 	})
-	.then(config => logger.log('Done. Optimised static files in', config.dest))
+	.then(output => {
+		const { fileSize, dest } = output;
+		filesizeReporter(fileSize, program.logLevel);
```

Comment:
I agree on the `filesize` / `fileSize` argument. However, the loglevel is temporarily. As soon as we make the logger a singleton, or the logger has the loglevel saved static, it can be removed here. The removal of the loglevel here will happen sooner than the addition of another argument. So lets leave it this way